og.module

```diff
@@ -210,12 +210,17 @@ function og_entity_create_access(AccountInterface $account, array $context, $bun
       continue;
     }
 
-    $handler = Og::getSelectionHandler($field_definition);
-
-    if ($handler->getReferenceableEntities()) {
+    $options = [
+      'target_type' => $field_definition->getFieldStorageDefinition()->getSetting('target_type'),
+      'handler' => $field_definition->getSetting('handler'),
+      'field_mode' => 'admin',
+    ];
+    /** @var \Drupal\Core\Entity\EntityReferenceSelection\SelectionPluginManager $handler */
+    $handler = \Drupal::service('plugin.manager.entity_reference_selection');
+
+    if ($handler->createInstance('og:default', $options)) {
```

Comment:
@kbrodej, the deprication says we should use SelectionPluginManager::getInstance. Why are we using createInstance() here?

Reply:
Oversight on my side. We should use SelectionPluginManager::getInstance() in all cases and we shouldn't consider createInstance() and getReferenceableEntities() as equal.